analytics-next/index.js

```diff
@@ -50,3 +50,20 @@ function setUserProperties() {
   setUserProperties(analytics, { favorite_food: 'apples' });
   // [END analytics_set_user_properties]
 }
+
+function recordScreenView() {
+  const appName = '<APP_NAME>';
+  const screenName = '<SCREEN_NAME>';
+
+  // [START analytics_record_screen_view]
+  const { getAnalytics, logEvent } = require("firebase/analytics");
+
+  const analytics = getAnalytics();
+  logEvent(analytics, 'screen_view', {
+    app_name: appName,
```

Comment:
@hsubox76 all 4 of these params seem to be required by the logEvent() type definition. Is that correct?

Reply:
I believe that's correct. I got the requirement for `app_name` and `screen_name` from the Analytics documentation [here](https://developers.google.com/analytics/devguides/collection/gtagjs/screens), and the other two are the ones we just discovered are required for Firebase to log the event.

Reply:
I think this is a mistake, because Android and iOS only require two user parameters:
https://firebase.google.com/docs/analytics/screenviews#kotlin+ktx

Reply:
You're right, after some digging around, that's the page for UA, this is the updated page for GA4: https://developers.google.com/analytics/devguides/collection/ga4/screen-view

And then for Firebase `screen_name` is replaced with `firebase_screen`. So you're correct, it should work with only the two parameters also used by Android and iOS.

Reply:
Thanks for digging! I will hold off on this PR until the types in the JS SDK are updated to only require the 2 params and then I'll update this and merge it.

Reply:
Published last Thursday, hopefully this works now.

Reply:
Thanks!